core/dfc.h

```diff
@@ -13,7 +13,7 @@
 #undef TRACE_SYSTEM
 #define TRACE_SYSTEM dfc
 #undef TRACE_INCLUDE_PATH
-#define TRACE_INCLUDE_PATH ../../../../vendor/qcom/opensource/datarmnet/core
+#define TRACE_INCLUDE_PATH ../../techpack/datarmnet/core
```

Comment:
If only this was passed via the makefile :grimacing:

Reply:
let it be for now

Reply:
Not saying that _we_ should, upstream should have written more composable things in the first place :wink: